### PR TITLE
Implement Github action for building static information database

### PR DESCRIPTION
**Pull request by pverscha**
This PR implements a Github workflow that automatically builds a database with all static information about GO-terms, EC-numbers, InterPro-entries and taxons, aimed at the Unipept Desktop application.

This workflow runs once every month and automatically bumps the version in the "workflows/static_database/version.txt" file, makes a new commit and tag and publishes a release with this DB-file. An example of such a release that was generated today can be seen [here](https://github.com/unipept/make-database/releases/tag/database-2020-05-12).

### CODE REVIEW

File: .github/workflows/static_database.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+      - name: Install required utilities
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install git make maven wget unzip expect gawk sqlite3 libsqlite3-dev
+      - name: Where is expect?
+        run: |
+          which expect
```
Comment (ninewise) [inline .github/workflows/static_database.yml:27]: This is some leftover debugging?

File: .github/workflows/static_database.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        run: |
+          make taxons
+          make functional_annotations
+      - name: Build SQLite database from generated files
+        shell: bash
+        run: |
+          chmod +x workflows/static_database/build_database.sh
+          ./workflows/static_database/build_database.sh
+      - name: ls
+        run: ls -hl
```
Comment (ninewise) [inline .github/workflows/static_database.yml:47]: This, too?

File: .github/workflows/static_database.yml (hunk 1 above)
Comment (pverscha) [reply .github/workflows/static_database.yml:27]: Oops, yes it is :)

File: .github/workflows/static_database.yml (hunk 2 above)
Comment (pverscha) [reply .github/workflows/static_database.yml:47]: This too :)

**Comment by pverscha** [comment]
@ninewise I removed the leftover debugging steps from the workflow

**Pull request merged**